### ISSUE TITLE
Virustotal flags installation files of v0.12 for windows as malicious

### ISSUE THREAD

**Issue by MaximilianRettinger** [ActivityWatch/activitywatch#797]
Hi,

I wanted to install activity watch on my new work machine running windows. Had good experience with AW so far on my linux machine.
Can you please clarify why this is happening? I would really like to use this great tool, but I simply can't install with red warning flags like this.

Zip:
![image](https://user-images.githubusercontent.com/112863328/190980163-cdda98e9-94c5-4511-a6f0-1687ab38d414.png)
Exe:
![image](https://user-images.githubusercontent.com/112863328/190980283-f7dbb979-0a5e-4191-9b85-6e32d8be0d80.png)

**Comment by VLEU** [comment]
This is definitely a false positive, and you should have no reasons to worry.

**Comment by MaximilianRettinger** [comment]
@VLEU Thanks for your reply. Doesn't really matter if I worry or not - it matters that the admin in the company that I am working for worries. 
So I'd have to provide some sort of proof to him, that this a false positive. Could you supply a valid reasoning to me why this is a false positive?

**Comment by VLEU** [comment]
VirusTotal simply aggregates the output of different antivirus vendors and URL scanners, it does not produce any verdicts of its own.  4/61 or 1/66 is definitely an indicator of a false positive.

When 4 obscure "virus scanners" that nobody uses and nobody has ever heard of say it contains a virus, and the other 100 true virus scanners, including all the big and reputable names say it is clean, that should tell them something.
A simple explanation for the false positive is that the 4 vendors might have some overly aggressive detection filters (or they simply suck - in layman's terms). 

Additionally, this entire project is open-source. If there was a problem, someone would most likely have noticed it and voiced some concern.

**Comment by florianklumb** [comment]
Handed in the 0.12.1 version to the Microsoft Security Intelligence portal and Windows Defender is now desensitized and will allow running the exe.

**Comment by MaximilianRettinger** [comment]
Thanks you both, that's more than sufficient for me!
